openy_node/modules/openy_node_blog/src/BlogSimplePaginator.php

```diff
@@ -40,28 +43,28 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager, Rou
    */
   private function buildQuery($direction = 'prev') {
     $currentNode = $this->routeMatch->getParameter('node');
-    
+
     if (!$currentNode) {
       return NULL;
     }
-    
+
     $query = $this->nodeStorage->getQuery();
 
     $compare = '>';
     $sort = 'ASC';
-    
+
     if ($direction == 'prev') {
       $compare = '<';
       $sort = 'DESC';
     }
 
     $result = $query->condition('created', $currentNode->getCreatedTime(), $compare)
-                 ->condition('type', $currentNode->bundle())
-                 ->condition('status', 1)
-                 ->condition('langcode', $currentNode->language()->getId())
-                 ->sort('created', $sort)
-                 ->range(0, 1)
-                 ->execute();
+      ->condition('type', $currentNode->bundle())
+      ->condition('status', 1)
+      ->condition('langcode', $currentNode->language()->getId())
+      ->sort('created', $sort)
+      ->range(0, 1)
```

Comment:
This query needs an access check as per [this change record](https://www.drupal.org/node/3201242).
```suggestion
      ->range(0, 1)
      ->accessCheck(FALSE)
```